doc/man7/EVP_RAND-TEST_RAND.pod

```diff
@@ -0,0 +1,88 @@
+=pod
+
+=head1 NAME
+
+EVP_RAND-TEST_RAND - The test EVP_RAND implementation
+
+=head1 DESCRIPTION
+
+A test random number generator which should never be used for production.
+It accepts seed material and regurgitates this verbatim instead of generating
+anything.
+
+=head2 Identity
+
+"TEST-RAND" is the name for this implementation; it
+can be used with the EVP_RAND_fetch() function.
+
+=head2 Supported parameters
+
+The supported parameters are:
+
+=over 4
+
+=item "strength" (B<OSSL_RAND_PARAM_STRENGTH>) <unsigned integer>
+
+Get the security strength of the test random number generator.
+This parameter cannot be set and will always have the value zero.
```

Comment:
Not sure the 2nd statement is needed.

Reply:
I'll leave it as "this param cannot be set".